ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -262,8 +262,22 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
           export TEST_PROVIDER='azure'
         elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+          set +x #make extra sure we aren't echo'ing these to stdout
+          export OS_CLOUD="openstack-cloud"
+          export OS_CLIENT_CONFIG_FILE="/tmp/cluster-profile/clouds.yaml"
```

Comment:
This should probably be made part of the container env, and maybe even all the way up to the template parameter section. 
soemthing like this: 
template:
  parameters:
     - name: OS_CLOUD
       value: openstack-cloud

  pods:
     container:
          env:
          - name: OS_CLOUD
            value: ${OS_CLOUD}